lua/config/lsp/setup.lua

```diff
@@ -139,5 +139,5 @@ require("mason-lspconfig").setup_handlers {
 
 require("ufo").setup({
   fold_virt_text_handler = ufo_config_handler,
-  close_fold_kinds = { "imports" },
+  close_fold_kinds_for_ft = { "imports" },
```

Comment:
```suggestion
  close_fold_kinds_for_ft = { default = {"imports", "comment"}, },
```
With your solution you get the same error as here: 
https://github.com/kevinhwang91/nvim-ufo/issues/209